lib/Basic/Default/TaskQueue.inc

```diff
@@ -100,10 +103,27 @@ bool TaskQueue::execute(TaskBeganCallback Began, TaskFinishedCallback Finished,
         T->Env.empty() ? decltype(Envp)(None)
                        : decltype(Envp)(llvm::toStringRefArray(T->Env.data()));
 
+    llvm::SmallVector<char, 64> stdoutPath;
+    llvm::SmallVector<char, 64> stderrPath;
```

Comment:
Ah, this is where to use SmallString, so that you don't need to explicitly make StringRefs later!